.github/workflows/docker.yml

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 Intel Corporation
+name: Docker image
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout and build
+    - uses: actions/checkout@v3
+    - name: Build and push Docker image
+      run: |
+        make docker-build
+
+  push-dockerhub:
+    env:
+      DOCKER_REGISTRY: "docker.io/"
+      DOCKER_REPOSITORY: "badhrinathpa/"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: echo GIT_SHA_SHORT=$(git rev-parse --short HEAD) >> $GITHUB_ENV
+
+      - id: docker-login
+        uses: docker/login-action@v2.1.0
+        with:
+          registry: docker.io
+          username: ${{ secrets.DOCKER_HUB_LOGIN }}
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+
+      - name: Build and push "master-latest" Docker images
+        env:
+          DOCKER_TAG: master-${{ env.GIT_SHA_SHORT }
```

Comment:
Do we get branch tag in the GitHub action ? As such this is ok for now, but want to know if we can use destination branch

Reply:
Yes we can add branch name in tag :  
      - name: Build
        run: docker build -t udm/tag-example:${GITHUB_REF##*/} .

Currently we are running for master only. so didnt add branch name.